### ISSUE TITLE
[BUG] Impossible to save content with an attribute image named image_1

### ISSUE THREAD

**Issue by pbnPierre** [bolt/bolt#6053]
Details
-------

 - Relevant Bolt Version: 3.2
 - Install type: Composer install
 - PHP version: 5.6 |
 - Used web server: Apache

Reproduction
------------
When you save a content with given structure in bolt 3.2 (With a database migrated From Bolt 2.X)
The Ajax save request edit content give a 500 error with this message :
Error 500 : An exception occurred while executing 'UPDATE bolt_guidebooks ...' SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'image_1' cannot be null
​
I think the problem come from the method __call in MagicAttributeTrait that does not support an image attribute with image_1 and try to call serializeImage1 and does not found an image1 attribute instead of an image_1 attribute

Here is the description in contenttype.yml

```
guidebooks:
    name: "Guides Pratiques"
    singular_name: "Guide Pratique"
    fields:
        title:
            label: Titre
            type: text
            class: large
            group: content
            required: true
        slug:
            type: slug
            uses: [ title ]
            group: content
        icon:
            label: Icône
            type: image
            group: media
            extensions: [ svg ]
        content:
            label: Contenu
            type: html
            height: 200px
            group: content
        title_1:
            type: text
            group: 'Feuillets'
            label: 'Titre Feuillet 1'
        image_1:
            type: image
            group: 'Feuillets'
            label: 'Image Feuillet 1'
        title_2:
            type: text
            group: 'Feuillets'
            label: 'Titre Feuillet 2'
        image_2:
            type: image
            group: 'Feuillets'
            label: 'Image Feuillet 1'
    searchable: true
    search_label: "Guides Pratiques"
    record_template: guidebook.twig
    listing_template: guidebooks.twig
    default_status: draft
    recordsperpage: 20
    icon_many: 'fa:book'
```

**Comment by pbnPierre** [comment]
Sorry for the contenttype.yml indentation

**Comment by pbnPierre** [comment]
Hi did someone try to reproduce this bug ?
And will someone try to fix it, I need to know if i must found an alternative solution.

**Comment by bobdenotter** [comment]
This seems to be the same issue as #6082, and should be fixed in #6088. 

Could you update to 3.2.3, and let us know if it's still a problem?

**Comment by pbnPierre** [comment]
Here is the result
![capture d ecran 2016-12-01 a 16 31 29](https://cloud.githubusercontent.com/assets/5654638/20799856/ca899722-b7e3-11e6-9f6f-d39665ed1b02.png)

**Comment by bobdenotter** [comment]
Pinging @rossriley: Is this something you know what's going on?

**Comment by pbnPierre** [comment]
It may be a loop on my event listeners

**Comment by pbnPierre** [comment]
It is a loop it works :) thanks

**Comment by GawainLynch** [comment]
Awesome, calling this fixed by #6088.
